Doc/library/shelve.rst

```diff
@@ -41,13 +41,24 @@ lots of shared  sub-objects.  The keys are ordinary strings.
    determine which accessed entries are mutable, nor which ones were actually
    mutated).
 
+   By default, :mod:`shelve` uses :func:`pickle.dumps` and :func:`pickle.loads`
```

Comment:
Let's mention that the `protocol` keyword may also be passed that specifies the pickle protocol.